packages/web/src/services/analytics.ts

```diff
@@ -0,0 +1,87 @@
+import Plausible from 'plausible-tracker';
+import ReactGA from 'react-ga';
+
+export type CustomEventProperties = {
+  label?: string;
+  value?: number;
+};
+
+let plausible: ReturnType<typeof Plausible>;
+
+function trackHistory() {
```

Comment:
Isn't there a way to plug into react-router? This seems to be quite a hackish approach.